### ISSUE TITLE
get into official mozilla add-on repository

### ISSUE THREAD

**Issue by dustymabe** [sciunto/exteditor#2]
would be great if we could get this plugin into the official add-on repo. The original author has had many requests for this over the years and it hasn't happened. Maybe there is a technical reason why, maybe not. 

Should we attempt to work with the original author one last time to get it into the repo? If that doesn't work maybe we can submit it ourselves?

**Comment by sciunto** [comment]
Indeed, I made this repository mainly for backup purposes. I do not have the knowledge to maintain a TB plugin. However, anyone who is interested can fork this repository.

**Comment by dustymabe** [comment]
@snipsnipsnip do you have those skills?

**Comment by eengstrom** [comment]
It's not hard.  The question is will they like the hack that was put in to add the layer of legacy support needed to quickly get the extension to run again.  

Longer term, we should just update the API that it targets for the preferences. 

I haven't dug in deeply, but it should be trivial.

**Comment by dustymabe** [comment]
@eengstrom - you up for the challenge? Would be grateful to you if you could make it happen. we can "skill swap" if there is something I can do that would help you.

**Comment by eengstrom** [comment]
I did a bit of reading and it may be we run into other issues quickly.  Notably, the "[Road to Firefox 57](https://blog.mozilla.org/addons/2017/02/16/the-road-to-firefox-57-compatibility-milestones/)" indicates that many extensions, including XUL extensions, are considered legacy and won't be supported (nor even loaded) post '57.  I think that `exteditor` might be an XUL add-on.  I'm not in the community deeply, so two questions - is it really using the XUL stuff (except in the manifest), and how will the TB roadmap differ, if at all?  I know they use many of the same underlying code base, so I'm going to guess that TB will follow suit, if not eventually, by default due to code base sharing.  

Anyway, I'll see what I can do - I've forked the repo myself, and I'm going to play, but it's not on my high priority list since it's working at the moment.

**Comment by dustymabe** [comment]
cool, i'll watch your repo. anything I can do to help (emotional support, beer money, etc), let me know.

Thanks!
